R/downsample.R

```diff
@@ -28,10 +28,7 @@
 #' @param target An integer that will be used to subsample. This
 #'  should not be set by the user and will be populated by `prep`.
 #' @param seed An integer that will be used as the seed when downsampling.
-#' @return An updated version of `recipe` with the new step
-#'  added to the sequence of existing steps (if any). For the
-#'  `tidy` method, a tibble with columns `terms` which is
-#'  the variable used to sample.
```

Comment:
For downsample and upsample the sentence for the `tidy()` method is now removed while in `newvalues.R` and `factor2string.R` the sentence was adapted and moved slightly. Is that on purpose?

Reply:
Yes, these have been moved to themis and are deprecated here in recipes, soon to go away here entirely.